CHANGELOG.rst

```diff
@@ -2,6 +2,22 @@
 Changelog
 =========
 
+0.19.0 (YYYY-MM-DD)
+===================
+
+* Now requires ``attrs >= 22.2.0``.
+
+* Added :class:`~.pipelines.DropLowProbabilityItemPipeline` that drops items
+  with the ``probability`` value lower than a set threshold.
+
+* Added the :attr:`.ListMetadata.validationMessages` attribute.
+
+* Reorganized the module structure.
+
+* Documentation improvements.
+
+* Test and CI fixes and improvements.
```

Comment:
There is a sneaky change in https://github.com/zytedata/zyte-common-items/pull/90/: new ListMetadata.get_date_downloaded_parsed method

Reply:
I've noticed it, but wasn't sure if it's intended to be used outside tests.